README.md

```diff
@@ -6,7 +6,7 @@ HTMLBars template helpers for additional truth logic in `if` and `unless` statem
 
 Helper   | JavaScript                           | HTMLBars
 ---------|--------------------------------------|-------------------
-eq       | `if (a === b)`                       | `{{if (eq a b)}}`
+eq       | `if ((a === b) === c)`               | `{{if (eq a b c)}}`
```

Comment:
To me this would read a bit easier: `a === b === c`. Do you think the paren are needed here to make it more clear?